frontend/desktop/src/services/backend/kubernetes.ts

```diff
@@ -35,10 +39,113 @@ export function K8sApi(config: string): k8s.KubeConfig {
       }
     });
   }
-
   return kc;
 }
 
+export async function getUser(uid: string) {
```

Comment:
Move detail function to spec ts files, do not put logic functions inside common base func file.